### PR TITLE
Add dropdown to account

### PR DESCRIPTION
**Pull request by smyzin**
- added dropdown to the account
- added `getEtherscanUrl` method

### CODE REVIEW

File: src/components/Navbar/AccountBlock.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    <AccentText color="text-accent">
+      <ENSAddress address={address} />
+    </AccentText>
+  )
+}
+
+export default function ({ address }: { address: string }) {
+  const { chain } = useNetwork()
+
+  const { disconnect } = useDisconnect()
```
Comment (T-Damer) [inline src/components/Navbar/AccountBlock.tsx:24]: Let's move this next to `useNetwork`

File: src/components/Navbar/AccountBlock.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  const onSelectOption = (selectedValue: string) => {
```
Comment (T-Damer) [inline src/components/Navbar/AccountBlock.tsx:25]: And let's add a blank line on top of this one

File: src/stores/AppStore.ts [hunk 3]
```diff
@@ -14,6 +14,12 @@ class AppStore {
   input?: ProofInput
   proof?: ECDSAProofStruct
   commitment?: bigint
+
+  resetOnDisconnect() {
+    this.connected = false
+    this.flowState = STATES.INIT
+    this.phase = this.input = this.proof = this.commitment = undefined
```
Comment (T-Damer) [inline src/stores/AppStore.ts:21]: Multiple assignment, cool :)

File: src/components/Navbar/AccountBlock.tsx [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+        chain &&
+        window.open(getEtherscanAddressUrl(address, chain.network), '_blank')
+    }
+  }
+
+  return (
+    <Dropdown
+      extraSpacing
+      fitToItemSize
+      currentValue={window.location.origin}
```
Comment (T-Damer) [inline src/components/Navbar/AccountBlock.tsx:40]: Should we use `useLocation` hook for this?

**Review by T-Damer — APPROVED** [review]

File: src/components/Navbar/AccountBlock.tsx (hunk 4 above)
Comment (smyzin) [reply src/components/Navbar/AccountBlock.tsx]: No need at all. Changed to address

**Review by T-Damer — APPROVED** [review]

**Pull request merged**